### DISCUSSION TITLE
暂停Mac端的官方安装包发布

### DISCUSSION THREAD

**Discussion by ChnMig** [TongchengOpenSource/AppScan#51] [Announcements]
经评估, 我们决定暂停Mac端的官方安装包发布, 原因如下:
- 开发者Mac设备损坏, 无法在Mac端进行Debug和功能支持
- MacOs针对软件运行有安全限制, 比如必须在有开发证书的设备进行打包才能在其他设备进行运行
- 经了解, AppScan的用户群体, 只有MacOs环境的用户特别少
- 开发者精力有限, 例如模拟器支持等功能, 两个平台的实现逻辑大相径庭, 影响新功能迭代速度

综上考虑, 我们决定暂停Mac端的官方安装包发布, 直到我们找解决办法
这并不代表目前的版本无法在Mac上运行和打包, 只是我们无法对新功能进行测试
MacOS用户, 可根据打包文档自行打包, 如您遇到问题, 可随时向我们反馈, 我们会提供力所能及的帮助.
您也可以参与到AppScan的开发中, 通过提交代码的方式修复MacOs的实测问题, 我们会积极接受来自用户的commit
谢谢各位对AppScan一如既往的支持!

**Comment by kuangren8** [comment]
能否添加ios app隐私合规检测的功能

**Comment by high0** [comment]
理解，支持团队和成员大佬们继续开发！期待
